yelp_package/Makefile

```diff
@@ -77,7 +77,7 @@ clean:
 VERSION = $(firstword $(subst -, ,$(RELEASE) ))
 # This sed isn't really right but at least it doesn't crash (and I couldn't get
 # \' or ''' or \\' to work :( )
-LAST_COMMIT_MSG = $(shell git log -1 --pretty=%B | sed -e "s/'/"/g")
+LAST_COMMIT_MSG = $(shell git log -1 --pretty=%B | sed -e 's/'\''/"/g')
```

Comment:
The fact that the string "ends" with the second single quote means that the string concatenation is handled by the shell itself. It works, but it's nasty. I'm going to replace this mess with \x27 which is less immediately readable but less ugly.